lib/NetworkManager.js

```diff
@@ -281,8 +285,9 @@ class Request {
    * @param {string} url
    * @param {string} resourceType
    * @param {!Object} payload
+   * @param {!Puppeteer.Frame} frame
```

Comment:
Frame is nullable here as well

Reply:
nice catch! thanks!
https://github.com/GoogleChrome/puppeteer/pull/1646/commits/31c31da50c65f8ced1cd7b49dc1e3b37bfb0bcf2